app/templates/adIndex.html

```diff
@@ -0,0 +1,242 @@
+  <fieldset>
+    <legend>Job Requirement</legend>
+    <div class="form-group">
+      <label for="inputSchoolName" class="col-md-2 control-label">School Name</label>
+      <div class="col-md-6">
+        <input class="form-control" id="inputSchoolName" placeholder="School Name" type="text" required>
+      </div>
+    </div>
+
+    <div class="form-group">
+      <label for="inputJobTitle" class="col-md-2 control-label">Job Title</label>
+      <div class="col-md-6">
+        <input class="form-control" id="inputJobTitle" placeholder="Job Title" type="text" required>
+      </div>
+    </div>
+
+    <!--div class="form-group">
+      <label for="inputSubject" class="col-md-2 control-label">Subject</label>
+      <div class="col-md-6">
+        <input class="form-control" id="inputSubject" placeholder="Subject" type="text">
+      </div>
+    </div-->
+
+    <div class="form-group">
+      <label for="inputSubject" class="col-md-2 control-label" required>Subject</label>
+      <div class="col-md-6">
+        <select class="form-control" id="inputSubject" required>
+          <option>SELECT SUBJECT</option>
+          <option>MATHEMATICS</option>
+          <option>ENGLISH LANGUAGE</option>
+          <option>BIOLOGY</option>
+          <option>ECONOMINS</option>
+        </select>
+     </div>
+    </div>
+
+
+    <!--div class="form-group">
+      <label for="inputSalaryRange" class="col-md-2 control-label">Salary Range</label>
+      <div class="col-md-6">
+        <input class="form-control" id="inputSalaryRange" placeholder="Salary Range" type="text">
+      </div>
+    </div-->
+
+
+    <div class="form-group">
+      <label for="Salary Range" class="col-md-2 control-label">Salary Range</label>
+      <div class="col-md-3">
+        <!--div class="col-md-12"-->
+        <input class="form-control" id="MinSalary" Placeholder="Min Salary" type="text" required>
+        <!--/div-->
+     </div>
+
+<script language="JavaScript">
+$(function(){
+  $('#MinSalary').keypress(function(e){
+    if(e.which == 48 || e.which == 49 || e.which == 50 || e.which == 51 || e.which == 52 || e.which == 53 || e.which==54 || e.which==55 || e.which==56|| e.which==57) {
+    } else {
+      return false;
+    }
+  });
+});
+</script>
+
+    <div class="col-md-3">
+        <input class="form-control" id="MaxSalary" Placeholder="Max Salary" type="text" required>
+     </div>
+    </div>
+<script language="JavaScript">
+$(function(){
+  $('#MaxSalary').keypress(function(e){
+    if(e.which == 48 || e.which == 49 || e.which == 50 || e.which == 51 || e.which == 52 || e.which == 53 || e.which==54 || e.which==55 || e.which==56|| e.which==57) {
+    } else {
+      return false;
+    }
+  });
+});
+</script>
+
+
+    <div class="form-group">
+      <label for="Experience" class="col-md-2 control-label">Years of Experience</label>
+      <div class="col-md-6">
+        <select class="form-control" id="Experience" required>
```

Comment:
What if the user has more than 5 years experience ?
Lets change this to be the following options:
1. Less than 1 year
2. 1 to 2 years
3. 3 to 5 years
4. 6 to 10 years
5. 11 years or more